src/Chainweb/Chainweb.hs

```diff
@@ -376,6 +380,9 @@ pChainwebConfiguration = id
     <*< configCutFetchTimeout .:: option auto
         % long "cut-fetch-timeout"
         <> help "After this many microseconds, give up trying to sync a particular Cut"
+    <*< configInitialCutHeightLimit .:: fmap (Just . BlockHeight) % option auto
```

Comment:
should this even be configurable? This sounds like best practice for catchup. We should poll our bootstraps for their most recent highest cut to determine this number on startup and defer to 0 otherwise.

Reply:
For this to be effective, we need an estimate of the height of the fork point. The better the estimate, the better the performance. 0 is always a valid estimate, but it's inefficient, because pact replay takes a long time -- in particular once we have transactions in the system.

To me it seems that those extreme scenarios with very deep forks are best resolved by manually switching to an "repair" mode. This options plus the `--private` flag would be such a "repair" mode.